gpg/helper.go

```diff
@@ -0,0 +1,85 @@
+package gpg
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"golang.org/x/crypto/openpgp"
+	"golang.org/x/crypto/openpgp/armor"
+	"golang.org/x/crypto/openpgp/packet"
+)
+
+// ReadEntity reads a single entity from a reader containing a list of entities.
+func ReadEntity(r io.Reader, armored bool) (*openpgp.Entity, error) {
```

Comment:
@v-maurer testing private functions should be ok according to https://talks.golang.org/2014/testing.slide#20